kafka/vendor/six.py

```diff
@@ -70,7 +70,9 @@ def __len__(self):
         else:
             # 64-bit
             MAXSIZE = int((1 << 63) - 1)
-        del X
+
+        # Don't del it here, cause with gc disabled this "leaks" to garbage
+        # del X
```

Comment:
Upstream doesn't have this fix, but I've filed a pull request: https://github.com/benjaminp/six/pull/176